Subject:  Fix 5.12-ck1 performance selection when "Default CPUFreq governor"
	  is chosen.
Patch Author: Orson Teodoro <orsonteodoro@hotmail.com>
Date: Jun 21, 2021 (Unix timestamp: 1624310174)

Problematic commit:
https://github.com/torvalds/linux/commit/70245442472f722670bf107cb4807c4d0f54f304

Error:

  GEN     modules.builtin
  LD      .tmp_vmlinux.kallsyms1
/usr/bin/x86_64-pc-linux-gnu-ld.bfd: drivers/cpufreq/cpufreq.o: in function `cpufreq_online':
cpufreq.c:(.text+0x41b7): undefined reference to `cpufreq_default_governor'
/usr/bin/x86_64-pc-linux-gnu-ld.bfd: drivers/cpufreq/cpufreq.o: in function `cpufreq_core_init':
cpufreq.c:(.init.text+0x9): undefined reference to `cpufreq_default_governor'
make: *** [Makefile:1208: vmlinux] Error 1

This build time error happens when performance is selected under the
"Default CPUFreq governor" setting is chosen but not when the ondemand is
chosen.

--- a/drivers/cpufreq/Kconfig.orig	2021-06-21 11:00:15.226548331 -0700
+++ b/drivers/cpufreq/Kconfig	2021-06-21 13:43:17.716335426 -0700
@@ -27,6 +27,9 @@ config CPU_FREQ_GOV_COMMON
 	select IRQ_WORK
 	bool
 
+config CPU_FREQ_DEFAULT_GOV_PERFORMANCE
+	bool
+
 config CPU_FREQ_STAT
 	bool "CPU frequency transition statistics"
 	help
